plugins/google-sheets/src/auth.ts

```diff
@@ -8,10 +8,11 @@ interface Tokens {
 
 interface StoredTokens {
     createdAt: number
+    /** Duration in seconds. */
     expiredIn: number
     accessToken: string
-    // When you have previously logged into the app with oAuth Google no longer gives a refresh token.
-    // https://stackoverflow.com/a/10857806
+    // After the initial login, Google does not provide new refresh tokens since
+    // the original does not expire. https://stackoverflow.com/a/8954103
```

Comment:
I think that because we now pass `prompt=consent` this is no longer the case and the value should technically always be there?

Reply:
😱

Reply:
Let me try

Reply:
Sorry I may be wrong, that would be the `Tokens` interface which already has that

Reply:
Just checked, yeah, it doesn't provide it on subsequent refreshes. We still need this